webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -48,6 +51,22 @@ export type Props = {
     footerDivider?: boolean;
     appendedContent?: React.ReactNode;
     headerButton?: React.ReactNode;
+    showCloseButton?: boolean;
+    showHeader?: boolean;
+
+    /*
+     * Controls the vertical location of the modal.
+     * 'top' => margin-top: 5vh
+     * 'center' => margin-top: calc(50vh - 240px)
+     * 'bottom' => margin-top: calc(50vh + 240px) (example calculation)
+     */
+    modalLocation?: ModalLocation;
```

Comment:
Similar comment here to the overridden padding in the InvitationModal. We should talk to UX to see if it's intentional that these are different since I think these would also ideally all be positioned the same in the app